iOS-sample-app/Showcases/Transactions/TransactionsViewController.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import ContentSquare
+
+struct MyItem
+{
+    private(set) var myTitle: String
+    private(set) var myPrice: Float
+    private(set) var myCurrency: Currency
+
+    init(title: String, price: Float, currency: Currency)
+    {
+        myTitle = title
+        myPrice = price
+        myCurrency = currency
+    }
+}
+
+// This class showcases how to track purchases made by an user with CustomerTransactions
+class TransactionsViewController: UIViewController {
+
+    private var itemsInCart = [MyItem]()
+    private var selectedCurrency : Currency = .eur
+    private var itemCount = 0
+
+    override func loadView()
+    {
+        super.loadView()
+
+        let validateButton = UIButton(type: .custom)
```

Comment:
Could you use standard iOS buttons? Such as the ones used in the modal screen views screen.